CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -374,6 +370,7 @@ void PuppiProducer::fillDescriptions(edm::ConfigurationDescriptions& description
   desc.add<double>("MinPuppiWeight", .01);
 
   PuppiAlgo::fillDescriptionsPuppiAlgo(desc);
+  PuppiContainer::fillDescriptionsPuppiContainer(desc);
```

Comment:
My review request was simply to remove from this class the local variables ```fPtMaxNeutrals``` and ```fPtMaxNeutralsStartSlope```, because they are not used here.

I'm not against to moving the definition of the parameters specific to PuppiContainer to the ```fillDescriptions``` method of that class, as you are doing here. However, if you do so you must do the same with all parameters specific to PuppiContainer, not only these two (there are a few more).

To keep the intervention at its minimum you could however simply revert to the previous behavior and have all parameters defined again inside PuppiProducer